LICENSE

```diff
@@ -0,0 +1,191 @@
+      any Contribution intentionally submitted for inclusion in the Work
+      by You to the Licensor shall be under the terms and conditions of
+      this License, without any additional terms or conditions.
+      Notwithstanding the above, nothing herein shall supersede or modify
+      the terms of any separate license agreement you may have executed
+      with Licensor regarding such Contributions.
+
+   6. Trademarks. This License does not grant permission to use the trade
+      names, trademarks, service marks, or product names of the Licensor,
+      except as required for reasonable and customary use in describing the
+      origin of the Work and reproducing the content of the NOTICE file.
+
+   7. Disclaimer of Warranty. Unless required by applicable law or
+      agreed to in writing, Licensor provides the Work (and each
+      Contributor provides its Contributions) on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+      implied, including, without limitation, any warranties or conditions
+      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+      PARTICULAR PURPOSE. You are solely responsible for determining the
+      appropriateness of using or redistributing the Work and assume any
+      risks associated with Your exercise of permissions under this License.
+
+   8. Limitation of Liability. In no event and under no legal theory,
+      whether in tort (including negligence), contract, or otherwise,
+      unless required by applicable law (such as deliberate and grossly
+      negligent acts) or agreed to in writing, shall any Contributor be
+      liable to You for damages, including any direct, indirect, special,
+      incidental, or consequential damages of any character arising as a
+      result of this License or out of the use or inability to use the
+      Work (including but not limited to damages for loss of goodwill,
+      work stoppage, computer failure or malfunction, or any and all
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2014 Red Hat, Inc.
```

Comment:
nit: I don't think this section is intended to be replaced (e.g. [Kubernetes does not alter it][1]).  It's just a template which you should alter when constructing your own boilerplate (which Kubernetes keeps in per-file headers like [this][2]).  If we don't want per-file headers, we may want the boilerplate statement in a `NOTICE` (like [the installer][3]) or in `README.md`.  Although I'm new to Red Hat, maybe there's a company position on this that I'm not aware of.

[1]: https://github.com/kubernetes/kubernetes/blob/v1.11.1/LICENSE#L190
[2]: https://github.com/kubernetes/kubernetes/blob/v1.11.1/pkg/api/endpoints/util.go#L2-L14
[3]: https://github.com/openshift/installer/blob/154d3b5aa0b9f66aea7106a348dc0f553e1ae273/NOTICE

Reply:
Huh, I hadn't even noticed that. You're right that the template shouldn't be modified. I'll actually download the license from the web this time.

Reply:
@abhinavdahiya I had to rebase the "first" commit (the license). _That_ is why you should always start a repo with an empty commit.